src/host.rs

```diff
@@ -417,6 +417,11 @@ pub fn matches(bind: SocketAddr, dst: SocketAddr) -> bool {
     bind == dst
 }
 
+/// Returns whether loopback is supported from src to dst
```

Comment:
Technically this is not a loopback. It ends up behaving like loopback because of how turmoil is routing to the local IP address. Mayeb `is_same`?